docs/guided-remediation.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: page
```

Comment:
nit: we should move the other experimental features into dedicated sub pages too for the same/consistent discoverability.